wireup/options.go

```diff
@@ -158,3 +158,10 @@ func WithHTTPClient(client *http.Client) Option {
 		builder.client = client
 	}
 }
+
+// WithLicense allows the caller to specify the subscription license (aka "track") they wish to use.
+func WithLicense(license string) Option {
+	return func(builder *clientBuilder) {
+		builder.license = license
```

Comment:
@DuncanBeutler - We need to make it possible for the caller to specify multiple licenses. (And you've already implemented the corresponding functionality in the `LicenseClient`.) So, let's transform `builder.license = license` into `builder.licenses = append(builder.licenses, license)`. That way, the caller can use the `WithLicense` option multiple times if desired.